src/_lib/sass/src/_popovers.scss

```diff
@@ -14,119 +15,166 @@
+    border-radius: 3px;
+    box-shadow: 0 0 8px rgba(0,0,0,.2);
+    font-family: 'Proxima Nova Regular', Arial, sans-serif;
+    z-index: 999;
+
     // Offset the popover to account for the popover arrow
     &.top     { margin-top: -$popover-arrow-width; }
     &.right   { margin-left: $popover-arrow-width; }
     &.bottom  { margin-top: $popover-arrow-width; }
     &.left    { margin-left: -$popover-arrow-width; }
-  }
-  
-  .popover-title {
+}
+
+//xavtodo: is it used anywhere?
+.popover-title {
     margin: 0; // reset heading margin
     padding: 8px 14px;
     font-size: $font-size-base;
     background-color: $popover-title-bg;
     border-bottom: 1px solid darken($popover-title-bg, 5%);
     border-radius: ($border-radius-large - 1) ($border-radius-large - 1) 0 0;
-  }
-  
-  .popover-content {
+}
+
+.popover-content {
     padding: 24px;
-  }
-  
-  // Arrows
-  //
-  // .arrow is outer, .arrow:after is inner
-  
-  .popover > .arrow {
-    &,
-    &:after {
-      position: absolute;
-      display: block;
-      width: 0;
-      height: 0;
-      border-color: transparent;
-      border-style: solid;
+
+    h5, p, .close{
+        color: get-color('gray');
+    }
+
+    h5{
+        //font-family: Proxima Nova SemiBold, Arial, sans-serif;
+        font-size: 16px;
+        margin-bottom: 8px;
     }
-  }
-  .popover > .arrow {
+
+    p{
+        font-size: 14px;
+        line-height: 17px;
+    }
+
+    .close {
+        position: absolute;
+        text-shadow: none;
+        opacity: 1;
+        //font-weight: 600;
+        font-size: 30px;
+        top: 3px;
+        right: 12px;
+        text-shadow: none;
+    }
+}
+
+// Arrows
+// .arrow is outer, .arrow:after is inner
+
+//overwrite default arrow border color;
+.popover.popover-top.bs-popover-top > .popover-arrow.arrow{
+    border-top-color: $popover-border-color;
+}
+
+.popover.popover-right.bs-popover-right > .popover-arrow.arrow{
+    border-right-color: $popover-border-color;
+}
+
+.popover.popover-bottom.bs-popover-bottom > .popover-arrow.arrow{
+    border-bottom-color: $popover-border-color;
+}
+
+.popover.popover-left.bs-popover-left > .popover-arrow.arrow{
+    border-left-color: $popover-border-color;
+}
+
+.popover > .arrow,
+.popover > .arrow:after{
+    position: absolute;
+    display: block;
+    width: 0;
+    height: 0;
+    border-color: transparent;
+    border-style: solid;
+}
+
+.popover > .arrow {
     border-width: $popover-arrow-outer-width;
-  }
-  .popover > .arrow:after {
+}
+
+.popover > .arrow:after {
     border-width: $popover-arrow-width;
     content: "";
-  }
-  
-  .popover {
-    &.top > .arrow {
-      left: 50%;
-      margin-left: -$popover-arrow-outer-width;
-      border-bottom-width: 0;
-      border-top-color: $popover-arrow-outer-fallback-color; // IE8 fallback
-      border-top-color: $popover-arrow-outer-color;
-      bottom: -$popover-arrow-outer-width;
-      &:after {
+}
+
+.popover.top > .arrow {
```

Comment:
Is there any reason why these styles have changed from 

```
.popover {
    &.top > .arrow {
```

to

`.popover.top > .arrow {`

Reply:
Just went through the styles and thought that nesting a lot of styles only to prepend the `.popover` in front of them when sass is compiled is of little value. Not nesting in that case offers improved legibility.